ambry-api/src/main/java/com/github/ambry/account/AccountBuilder.java

```diff
@@ -228,8 +228,10 @@ public Account build() {
     }
     // Did we check the container parent account id here?
     for (Map.Entry<Short, Container> entry : idToContainerMetadataMap.entrySet()) {
-      if (entry.getValue().getParentAccountId() == Container.UNKNOWN_CONTAINER_PARENT_ACCOUNT_ID) {
+      if (entry.getValue().getParentAccountId() == Container.UNKNOWN_CONTAINER_ID) {
```

Comment:
nit: In terms of value this might work fine, because we map both `UNKNOWN_CONTAINER_ID` and `UNKNOWN_CONTAINER_PARENT_ACCOUNT_ID` to same values. But logically, they can be different.

Since we aren't getting rid of `UNKNOWN_CONTAINER_PARENT_ACCOUNT_ID` , we don't need to change here.